src/commands/build/build.ts

```diff
@@ -8,6 +7,9 @@ import {
   createFile,
   asyncForEach
 } from '@sasjs/utils'
+import { Target, ServerType } from '@sasjs/utils/types'
+import { StreamConfig } from '@sasjs/utils/types/config'
+import { ServerTypeError } from '@sasjs/utils/error'
```

Comment:
Can we not import all from '@sasjs/utils' ?

Reply:
fixed